guides/users/users/overview.md

```diff
@@ -0,0 +1,78 @@
+# Overview
+
+Solidus includes a straightforward user management system. Users are your stores
```

Comment:
I think we can remove "straightforward" here or replace it with "sophisticated". But leaving it out is most likely the best option.